config/config_partial.go

```diff
@@ -457,11 +457,17 @@ func PartialParseConfig(ctx *ParsingContext, file *hclparse.File, includeFromChi
 			}
 		case TerragruntInputs:
 			allControls := ctx.TerragruntOptions.StrictControls
-			skipDependenciesInputs := allControls.FilterByNames(controls.SkipDependenciesInputs)
-			logger := log.ContextWithLogger(ctx, ctx.TerragruntOptions.Logger)
 
+			skipDependenciesInputs := allControls.Find(controls.SkipDependenciesInputs)
+			if skipDependenciesInputs == nil {
+				return nil, errors.New("failed to find control " + controls.SkipDependenciesInputs)
+			}
+
+			skipDependenciesInputs.SuppressWarning()
+
+			logger := log.ContextWithLogger(ctx, ctx.TerragruntOptions.Logger)
 			if err := skipDependenciesInputs.Evaluate(logger); err != nil {
```

Comment:
```suggestion
			logger := log.ContextWithLogger(ctx, ctx.TerragruntOptions.Logger)
			if err := skipDependenciesInputs.SuppressWarning().Evaluate(logger); err != nil {
```

Reply:
See [this](https://github.com/gruntwork-io/terragrunt/pull/3879#discussion_r1954901866) about keeping things consistent with enable.